Создан экран Избранное

manualcheg opened the pull request:
1. Создана разметка экрана
2. Добавлены некоторые новые размеры
3. Создан класс FavouritesFragment
4. Создан класс FavouritesViewModel

manualcheg committed:
1. Создана разметка экрана
2. Добавлены некоторые новые размеры
3. Создан класс FavouritesFragment
4. Создан класс FavouritesViewModel

manualcheg committed:
1. LinearLayout плейсхолдера сделан видимым

BigMag74 commented on app/src/main/res/layout/fragment_favourites.xml:
```diff
@@ -0,0 +1,67 @@
+        android:layout_marginEnd="@dimen/padding_horizontal_small"
+        android:fontFamily="@font/ys_display_medium"
+        android:gravity="center_vertical"
+        android:text="@string/vacancy_search"
```
Текст в заголовке от экрана Поиска

BigMag74 replied:
и нужно id поменять со snake_case на camelCase

anonim-no replied:
и добавить тип элемента

manualcheg replied:
Поправил

manualcheg committed:
1. Исправлены недочёты:
- убраны match_parent там, где нужно
 - заменена константа в заголовке на верную
 - переименованы id элементов согласно договоренности

manualcheg committed:
1. Исправлены недочёты:
- убран параметр android:orientaition в ConstraintLayout

anonim-no committed:
Merge branch 'dev' into feature/TECH-29/create_favourites_layout
